src/channel.rs

```diff
@@ -530,7 +540,7 @@ impl Channel {
     }
 
     /// Create a call using the method and option.
-    pub fn create_call<P, Q>(&self, method: &Method<P, Q>, opt: &CallOption) -> Result<Call> {
+    pub(crate) fn create_call<Req, Resp>(&self, method: &Method<Req, Resp>, opt: &CallOption) -> Result<Call> {
```

Comment:
@BusyJay PTAL

Reply:
does clippy show that we need to add `pub(crate)` here?

Reply:
clippy does not check pub exports

Reply:
so how can we know to add crate here?

Reply:
based on the function functionality and the generated server / client code, it is not used publicly.